src/intTest/resources/application.yml

```diff
@@ -4,6 +4,8 @@ server:
 logging:
   level:
     uk.nhs.digital.nhsconnect.lab.results: ${LAB_RESULTS_LOGGING_LEVEL:DEBUG}
+  pattern:
```

Comment:
I can see the logging pattern has been applied to unit and integration tests, but should this also be applied to the application `main`?

Reply:
I was thinking about that as well, but just in case followed the same logic as in NHAIS because they don't have it there. However, I also think that having this pattern in main shouldn't damage us. Should I add it?

Reply:
Consider asking @mattd-kainos if there's a reason it's missing on NHAIS. Observability is very important, but it could just be that the reality of the live service is that they don't need it, but it remains useful for debugging pre-prod work.

Reply:
I've found it here: https://github.com/nhsconnect/integration-adaptor-nhais/blob/develop/src/main/resources/application.yml#L10

Reply:
Oh, they actually do have it. Not sure why I thought they don't have it and how I didn't spot it. I guess it means that we definitely need it. My bad. Will fix now. Thanks!

Reply:
Fixed in c4be23ef47fe41e49e150221c87f24a312584c94. Thanks!

Reply:
Just a suggestion, we could improve the logging statements by removing `ConversationId=` when there's no conversation id, see http://logback.qos.ch/manual/layouts.html#replace. Thoughts?

Reply:
As the Acceptance Criteria says: "THEN every log entry created should contain Correlation-Id value". However, the suggestion is very good! Thanks for that!

@bartek-sarul what do you think about that, should we change Acceptance Criteria to implement it or should we leave it as it is?